### ISSUE TITLE
obtain the log probabilities of output tokens

### ISSUE THREAD

**Issue by Yuxin916** [BAAI-DCAI/Bunny#126]
Hi! Thank you for providing this framework. I found it very useful and helpful! 

I am new to VLM domain and currently using SigLip vison encoder and Phi-2 LLM as main components on my task. I was wondering how can i obtain the log probabilities of output tokens after i called model.generate()? Do you have any suggestions on how to call on or where could i develop on this? 

Best Regards

**Comment by Isaachhh** [comment]
There is a related issue https://github.com/BAAI-DCAI/Bunny/issues/62.

And you may refer to [logits](https://huggingface.co/docs/transformers/internal/generation_utils#transformers.generation.GenerateDecoderOnlyOutput.logits).

**Comment by Yuxin916** [comment]
Thank you! The problem is solved.
